java/pom.xml

```diff
@@ -9,6 +9,13 @@
     <build>
         <plugins>
             <plugin>
+                <artifactId>maven-compiler-plugin</artifactId>
```

Comment:
This change is fine since it lets you conpile with 1.5 and up. It just means any version >=5.